cmd/build/helmify/static/values.yaml

```diff
@@ -38,6 +38,12 @@ postUpgrade:
       pullPolicy: IfNotPresent
       pullSecrets: []
     extraNamespaces: []
+    podSecurity: ["pod-security.kubernetes.io/audit=restricted",
+      "pod-security.kubernetes.io/audit-version=latest",
+      "pod-security.kubernetes.io/warn=restricted",
+      "pod-security.kubernetes.io/warn-version=latest",
+      "pod-security.kubernetes.io/enforce=restricted",
+      "pod-security.kubernetes.io/enforce-version=v1.24"]
```

Comment:
Should we get it via `Capabilities.KubeVersion.Version`

Reply:
we know we are compatible with 1.24, but we won't know it for future versions. That's why enforce is pinned while audit and warn is latest